src/cc_toolchain/wrapper/arm-none-eabi-g++

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3 -B
+
+# Copyright 2018 Josh Pieper, jjp@pobox.com.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+
+"""Delegate to the C compiler and strip C++ specific options if
```

Comment:
Curious why we're delegating to the C compiler and effectively compiling this like C if this is supposed to be for g++?

Edit: Oh I see this isn't entirely true, we only strip and delegate _if_ there are `.cpp` files. A more clear comment would be useful.

Reply:
bump

Reply:
Changed to `"""Delegate to the C compiler and strip C++ specific options if we're compiling C files"""`

Reply:
How about
`"""Delegate to the C compiler and strip C++ specific options if we are not compiling any .cpp` files"""`

And can we move this comment down into the `main` function where it's more relevant? I don't think it really belongs at the top of the file.

Reply:
Agreed. Changed and moved.

Reply:
We should be able to resolve this now?